wp-includes/class-wp-offline-page.php

```diff
@@ -0,0 +1,74 @@
+<?php
+/**
+ * WP_Offline_Page class.
+ *
+ * @package PWA
+ */
+
+/**
+ * This class manages the Offline Page.
+ */
+class WP_Offline_Page {
```

Comment:
This class is pretty small. Would it make more sense to have `WP_Offline_Page_UI` and `WP_Offline_Page_Excluder` be just integrated into this one class? It could be called `WP_Offline_Manager` perhaps.

Reply:
That sounds fine with me. We can always revisit later if it makes sense to merge into a single class for the core merge. But for now we can continue as it is.